tests/bsim/bluetooth/host/security/unpairing_security_changed/src/central.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+
+#include <zephyr/logging/log.h>
+#include <zephyr/bluetooth/addr.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/bluetooth.h>
+
+#include "bs_bt_utils.h"
+
+LOG_MODULE_REGISTER(test_central, LOG_LEVEL_DBG);
+
+BUILD_ASSERT(CONFIG_BT_BONDABLE, "CONFIG_BT_BONDABLE must be enabled by default.");
+
+void central(void)
```

Comment:
Where does the central disconnect after pairing has completed?

Reply:
Remember it is the central that disconnects right after pairing completes in the description of issue #61465.

Reply:
The behavior of the Peripheral is the same as the Central, as far as the `pairing_complete` callback is concerned. Have you tested this patch on your platform? We think that it does fix the issue.

We can't test the Central in the same way, as the Central will call the security changed callback _before_ distributing the keys: the Central will not send its keys until it has received the Peripheral's.

Reply:
Ok. It just makes it a more unclear that the test is testing the steps to reproduce in the issue description. But ok, if it creates the same situation.

I tested the smp.c patch on our platform, and it works :-)